includes/multi-currency/MultiCurrency.php

```diff
@@ -1105,6 +1184,92 @@ private function register_admin_scripts() {
 		);
 	}
 
+	/**
+	 * Gets the multi currency onboarding preview overrides from the querystring.
+	 *
+	 * @return  array  Override variables
+	 */
+	public function get_multi_currency_onboarding_simulation_variables() {
+		// Check if we are in a preview session, don't interfere with the main session.
+		if ( ! isset( $_GET['is_mc_onboarding_simulation'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			return [];
+		}
+
+		// Define variables which can be overridden inside the preview session, with their sanitization methods.
+		$possible_variables = [
+			'enable_storefront_switcher' => 'wp_validate_boolean',
+			'enable_auto_currency'       => 'wp_validate_boolean',
+		];
+
+		// Define the defaults if the parameter is missing in the request.
+		$defaults = [
+			'enable_storefront_switcher' => false,
+			'enable_auto_currency'       => false,
+		];
+
+		// Prepare the params array.
+		$values = [];
+
+		// Walk throught the querystring parameter possibilities, and prepare the params.
```

Comment:
I tend to write it wrong too, it's `through` rathen than `throught` :)

Reply:
Thanks :) I committed your suggestion!